manifest/spark-operator-rbac.yaml

```diff
@@ -67,3 +67,22 @@ roleRef:
   kind: ClusterRole
   name: sparkoperator
   apiGroup: rbac.authorization.k8s.io
+---
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRole
+metadata:
+  labels:
+    rbac.authorization.k8s.io/aggregate-to-admin: "true"
+  name: sparkoperator-aggregate-to-admin
+rules:
+- apiGroups: ["sparkoperator.k8s.io"]
```

Comment:
We actually have a `ClusterRole` above covers `sparkapplications` and `scheduledsparkapplications`. See line 54 above.

Reply:
I see. I guess the case if you are managing for example RBACs for teams, users, service accounts, etc, and you attach a an `admin` role in specific namespace for them. Then you expect to have admin access for most of namespace resources except CRDs like sparkapplications, scheduledsparkapplications.

Aggregate that specific role in the PR will add to the admin role (which is defined by default in kubernetes) the ability to do `sparkapplications`, `scheduledsparkapplications`.

Reply:
We just got that use case, because we give (for example) users in some specific namespaces  admin role, but they still can't use `sparkapplications`, `scheduledsparkapplications`. By aggregate that role to admin, any admin will be able to use them.

Reply:
Got it. So what the changes here do is they extend the `cluster-admin` to also include full permissions to manipulate `sparkapplications` and `scheduledsparkapplications`, right?

Reply:
Yes, specifically role `admin` which have some predefined permissions (not including `sparkapplications` and `scheduledsparkapplications`). We usually use this role (`admin`) for granting admin access for namespace level. But the one you have mentioned `cluster-admin` already has `[*]` for resources and verbs; there is no need to aggregate spark resources to it.

Reply:
Got it. Thanks.